zipkin-lens/src/components/TracePage/TraceSummary.jsx

```diff
@@ -11,46 +11,78 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useMemo } from 'react';
 import Box from '@material-ui/core/Box';
-// import { AutoSizer } from 'react-virtualized';
+import { AutoSizer } from 'react-virtualized';
```

Comment:
just a minor thing :D I think we can upgrade to react-window soon!

Reply:
Thank you for your information!